src/App.js

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import AppHeader from './components/AppHeader';
 
 const App = () => {
   console.log(chatMessages);
 
   return (
     <div id="App">
-      <header>
-        <h1>Application title</h1>
+      <header id="App header">
+        <AppHeader messages={chatMessages} />
```

Comment:
Very creative that you created a new component to deal with the header and made it adaptable for the future cases  with more than two users, etc.